Mlem/API/APIClient/APIClient.swift

```diff
@@ -80,8 +80,8 @@ class APIClient {
         switch flow {
         case let .account(account):
             session = .authenticated(account.instanceLink, account.accessToken)
-        case .onboarding:
-            // no calls to our `APIClient` should be made during onboarding
+        default:
+            // no calls to our `APIClient` should be made during onboarding or applock
```

Comment:
I think this can be reverted?

```suggestion
        case .onboarding:
            // no calls to our `APIClient` should be made during onboarding
```